resources/views/external-link-confirm.blade.php

```diff
@@ -6,61 +6,81 @@
 
 @php($identifier = md5($url . '-' . Str::random(8)))
 
-<a
-    href="javascript:;"
-    x-data="{
+<div x-data="{
         openModal() {
             Livewire.emit('openModal', '{{ $identifier }}')
+        },
+        redirect() {
+            window.open('{{ $url }}', '_blank')
+        },
+        hasConfirmedLinkWarning() {
+            return localStorage.getItem('confirm_link_warning') == 'true';
+        },
+        hasDisabledLinkWarning() {
+            return localStorage.getItem('disable_link_warning') == 'true';
```

Comment:
what's the difference :susp:

Reply:
As sammie asked, need two confirmations basically, only checking the checkbox isn't enough, you need to check the box and then click on the button to follow the link for your preference to be saved.

So `confirm_link_warning` -> checked box
`disable_link_warning` -> Clicked the button

Reply:
see slack, the checkbox state should not be kept between modal instances @Highjhacker , so you can remove one of these

Reply:
Looking into that, was easy to handle with localStorage cause no issues with accessing parent variables, but here since that checkbox is within another alpine component (`x-ark-js-modal`) it's a bit more annoying, but I'll figure out something